### PR TITLE
[admin_v3] New brand colours in admin interface

### PR DESCRIPTION
**Pull request by dacook**
### What? Why?

- Part of #10630

Setting up a new feature-toggled stylesheet, and just getting started on BUU with some colour changes. As you can see, there is clearly more to do.

There are some minor changes of reds that will appear outside of feature toggle.

1. **Change error red, warning red and warning orange, all to red** ([ref](https://github.com/openfoodfoundation/openfoodnetwork/issues/10630#issuecomment-1550511778))
    * admin_v3: Mojo `#C85136`
    * admin: `#C60F13`
Screen Shot 2023-05-17 at 12 10 22 pm

In admin_v3, the error messages will need adjusting: 
Screen Shot 2023-05-26 at 3 30 43 pm

2. **[admin_v3] Update Spree blue to Allports `#008397`**
Screen Shot 2023-05-26 at 3 20 39 pm

Don't worry, we'll continue to work on the admin_v3 styles!!

### What should we test?

*Existing Admin interface*
Error messages (eg invalid form input) and icons will have a slightly different red, but everything else should look the same. The warning icon styles have been refactored slightly, so
* Check that the icons appear on `/admin/enterprise` 
* Click on enterprise fees, and try adding a fee without a calculator. Error message should appear above
* Click on tax categories, and try to edit and remove a name and save. Error message should appear.

*Admin v3 styles*
* Enable feature toggle at `/admin/feature-toggle/features/admin_style_v3`
* Observe new colours, but everything else still works.
* Repeat above tests.

### Release notes

Changelog Category: User facing changes > 😎 Feature toggled

This does slightly change some warning colours for all users, but it doesn't seem worth announcing.

### Dependencies
- https://github.com/openfoodfoundation/openfoodnetwork/pull/10779

### CODE REVIEW

**Review by jibees — APPROVED** [review]
Clear, perfect. 

I personaly find the contrast on the selected top tab not so easy to read:
Capture d’écran 2023-05-26 à 10 03 57

Therefore, I tested its contrast and accessibility : https://coolors.co/contrast-checker/ffffff-9fc820
and it's not that good: 

Capture d’écran 2023-05-26 à 10 05 46

File: app/assets/javascripts/admin/utils/services/status_message.js.coffee [hunk 1]
```diff
@@ -5,8 +5,8 @@ angular.module("admin.utils").factory "StatusMessage", ->
       alert:    {style: {color: 'grey'}}
       notice:   {style: {color: 'grey'}}
       success:  {style: {color: '#9fc820'}}
-      failure:  {style: {color: '#da5354'}}
-      error:   {style: {color: '#da5354'}}
+      failure:  {style: {color: '#C85136'}}
+      error:   {style: {color: '#C85136'}}
```
Comment (mkllnk) [inline app/assets/javascripts/admin/utils/services/status_message.js.coffee:9]: I wonder if we could define CSS variables... but probably not worth it at the moment. So this does change some colours even without the feature toggle on.

**Review by mkllnk — APPROVED** [review]
Nice work. Just to understand: Assigning red to warnings is done generally but the type of red is changed only with the feature toggle. This means that some AngularJS templates have a different red to the rest now.

**Comment by filipefurtad0** [comment]
Hey @dacook ,

Thanks for the notes on what to test :pray: 
Following these:

- Check that the icons appear on /admin/enterprise

**Before this PR**
image

**After this PR**
image


**After enabling  _Admin v3 styles_**

image


- Click on enterprise fees, and try adding a fee without a calculator -> this triggers a known error 500; I guess you mean such an error message:

**Before this PR**
image

**After this PR**
image

After enabling  _Admin v3 styles_

image


- Click on tax categories, and try to edit and remove a name and save. Error message should appear.

**Before this PR**
image

**After this PR**
image

After enabling  _Admin v3 styles_

image

Noticed no change in functionality. Looks good to me!
Merging.

**Pull request merged**